### ISSUE TITLE
RFE: Add optional costs to scenarios

### ISSUE THREAD

**Issue by sldfgunslinger2766** [MegaMek/mekhq#1814]
Please consider making a change to Loot.java to allow negative integers. 
Loot.java line 113:
        if(cash.isPositive()) {
                desc += cash.toAmountAndSymbolString();

Can this be changed to also allow/check for a negative amount? This would allow the addition of a flat CB 'cost' for a player generated Scenario in the form of a checkbox to be enabled during manual resolution of the Scenario that applies a negative integer to a units 'Cash'. Thank you for your time and consideration!

**Comment by sldfgunslinger2766** [comment]
Edit: I think this is possibly a more elegant solution that accomplishes the intent of the original RFE with fewer additions/changes.

**Comment by AaronGullickson** [comment]
Yes, if the goal is to add a flat cost (ala warchest system, perhaps?), that should probably just be implemented as a separate thing.

**Comment by HammerGS** [comment]
This posted in Channel, with this comment - That was exactly what I was thinking.
![image](https://user-images.githubusercontent.com/13752452/85210960-d2ba8600-b301-11ea-9dde-bac3ad6b6267.png)

**Comment by sldfgunslinger2766** [comment]
Thank you, Hammer.

**Comment by sldfgunslinger2766** [comment]
> Yes, if the goal is to add a flat cost (ala warchest system, perhaps?), that should probably just be implemented as a separate thing.

Actually, this ticket can be moved to skipped/closed. You are correct that this will need to be implemented separately. This attempt at a work around won't really work as WP>SP>CB conversion takes place post game. Trying to 'front load' Missions via a flat CB cost doesn't really work.

**Comment by AaronGullickson** [comment]
I think we can leave it up, with a change to the title. It could be implemented something like the image from @HammerGS . That would make it easier to integrate later if we every get warchest system and/or story arcs implemented.

**Comment by HammerGS** [comment]
> I think we can leave it up, with a change to the title. It could be implemented something like the image from @HammerGS . That would make it easier to integrate later ~~if we~~ when we ever get warchest system and/or story arcs implemented.

:)

**Comment by IllianiCBT** [comment]
This appears to already be implemented. Closing.
